README.md

```diff
@@ -2,4 +2,9 @@
 
 ![Integration Test Action](https://github.com/HAECHI-LABS/henesis-wallet-sdk/workflows/Integration%20Test%20Action/badge.svg) [![Build Status](https://travis-ci.com/HAECHI-LABS/henesis-wallet-sdk.svg?branch=master)](https://travis-ci.com/HAECHI-LABS/henesis-wallet-sdk)
 
+## How to Build
 
+```shell
+npm run bootstrap
+npm run build
```

Comment:
```suggestion
npm run bootstrap
```
`npm run bootstrap` includes `npm run build`, so only the `npm run bootstrap` command is enough.

Reply:
Thanks!